### ISSUE TITLE
Local customization for "regular expressions"

### ISSUE THREAD

**Issue by gordonzola** [WitchesTown/mastodon#12]
Mastodon web UI allows witches and other kind of people to save regexes (some weird magic from people who talk to computers) in order to filter the home timeline. We, as very cool witches, should have a better term for this.

I suggest using “cast a repellent spell” instead of "filter out by regular expressions”, or, as the cheese-eaters (and hummus-eaters, but speaking french) say, “lancer un sort d’éloignement”

* * * *

- [ x ] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [ x ] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by Sigafoos** [comment]
I agree that regular expressions are the closest thing we have to tangible witchcraft, but I'm worried that "cast a repellent spell" is clever but would make the functionality confusing. I mean, regex in general is probably a confusing topic as it is...

But then again, plain text filtering works, so that might be sufficient. Or "banish jinxes with...", or something of that sort.

And to be clear, I am fully on board with renaming it!

**Comment by gordonzola** [comment]
This feature is being rewritten upstream anyway, so we could chose a new label now, and reopen this issue when the new filtering system will be released.
